charts/seed-monitoring/charts/grafana/dashboards/operators/apiserver-watches-dashboard.json

```
@@ -0,0 +1,391 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 1,
+  "id": 22,
+  "iteration": 1597754908762,
+  "links": [],
+  "panels": [
+    {
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 4,
+      "panels": [],
+      "repeat": "Scope",
+      "scopedVars": {
+        "Scope": {
+          "selected": true,
+          "text": "cluster",
+          "value": "cluster"
+        }
+      },
+      "title": "Watch Durations for Scope: $Scope",
+      "type": "row"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "description": "$Quantile of watches on this resource are less than or equal to the value.",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {}
+        },
+        "overrides": []
+      },
+      "fill": 1,
```

I think `fill=0` makes this dashboard easier on the eyes 😉 
```suggestion
      "fill": 0,
```